lib/compiler-finder.ts

```diff
@@ -372,14 +403,17 @@ export class CompilerFinder {
     }
 
     getExes() {
-        const langToCompilers = this.compilerProps(this.languages, 'compilers', '', exs => _.compact(exs.split(':')));
+        const langToCompilers = this.compilerProps(this.languages, 'compilers', '', exs =>
+            _.compact((exs as string).split(':')),
+        );
         this.addNdkExes(langToCompilers);
         logger.info('Exes found:', langToCompilers);
         return langToCompilers;
     }
 
     addNdkExes(langToCompilers) {
-        const ndkPaths = this.compilerProps(this.languages, 'androidNdk');
+        // TODO: Fix types here
+        const ndkPaths = this.compilerProps(this.languages, 'androidNdk') as unknown as _.Dictionary<any>;
```

Comment:
How come this needs to be a _.Dictonary?

Reply:
Good question, I think it's actually a `Record<string, string>`